### PR TITLE
Fix request logger error for delayed response

### PR DESCRIPTION
**Pull request by miherlosev**
@AndreyBelym

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
Don’t review it. I will to check an another way to fix problem

**Comment by miherlosev** [comment]
Waiting for fix problem with Node 10

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR @AndreyBelym

File: test/server/request-hooks-test.js [hunk 1]
```diff
@@ -181,6 +183,24 @@ describe('RequestLogger', () => {
...
+    it('.contains and .count methods should affect only completed requests', () => {
+        const logger            = new RequestLogger();
+        const requestEventMock2 = createProxyRequestEventMock(nanoid(9), nanoid(9));
+
+        logger.onRequest(requestEventMock);
+        logger.onResponse(responseEventMock);
+        logger.onRequest(requestEventMock2);
+
+        return Promise.all([
+            logger.count(r => r.request),
```
Comment (AndreyBelym) [inline test/server/request-hooks-test.js]: Could you please check that requests have the `response` property?

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Pull request merged**